test/system/280-update.bats

```diff
@@ -309,4 +309,22 @@ function nrand() {
 
     run_podman rm -t 0 -f $ctrname
 }
+
+# bats test_tags=ci:parallel
+@test "podman update - check resources over write Resources configuration" {
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname \
+                --pids-limit 1024     \
+                $IMAGE /home/podman/pause
+
+    run_podman update $ctrname --memory 100M
+
+    run_podman inspect $ctrname --format {{.HostConfig.Memory}}
+    assert "$output" == "104857600" ".HostConfig.Memory"
+
+    run_podman inspect $ctrname --format {{.HostConfig.PidsLimit}}
```

Comment:
Add a link to the bug https://github.com/containers/podman/issues/24610 here and add a sentence to explain this pid check is done to ensure other resource setting are not unset.